src/components/widgets/DefaultWidget.tsx

```diff
@@ -3,22 +3,38 @@ import { StyleSheet } from 'react-native';
 import { TouchableOpacity } from 'react-native-gesture-handler';
 
 import { IconProps, normalize } from '../../config';
+import { Image } from '../Image';
 import { BoldText, RegularText } from '../Text';
 import { WrapperRow, WrapperVertical } from '../Wrapper';
 
 type Props = {
   count?: number | string;
   Icon: (props: IconProps) => JSX.Element;
+  image?: {
+    height?: number;
+    url: string;
+    width?: number;
+  };
   onPress: () => void;
   text: string;
 };
 
-export const DefaultWidget = ({ Icon, count, onPress, text }: Props) => {
+export const DefaultWidget = ({ Icon, count, onPress, text, image }: Props) => {
   return (
     <TouchableOpacity onPress={onPress}>
       <WrapperVertical style={styles.container}>
         <WrapperRow center>
-          <Icon style={[styles.iconWithoutCount, !!count?.toString() && styles.iconWithCount]} />
+          {image ? (
```

Comment:
`image` is not boolean, so !! please

Reply:
i suggest checking the url `!!image?.url` as this is required to render an image

Reply:
I didn't use `!!` because using it like this causes an Eslint error. But yes you are right we should check `image?.url`.
<img width="380" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/58ec18c3-f218-4400-8ebc-7463e9221fd9">

Reply:
I added `image?.url` control.

https://github.com/smart-village-solutions/smart-village-app-app/commit/30ddb5e86e2a4a1b2abc6ba74ca80a359c6df851

Reply:
ok because we check with ? and not with &&